Example/CardParts/CardPartIconLabelCardController.swift

```diff
@@ -10,6 +10,7 @@ import Foundation
 import CardParts
 import RxSwift
 import RxCocoa
+import RxGesture
```

Comment:
@badrinathvm Where are we using `RxGesture`?

Reply:
Not required any more. removed.